upstream-community-operators/camel-k/1.3.0/camel-k.v1.3.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,573 @@
+    mediatype: image/svg+xml
+  install:
+    spec:
+      clusterPermissions:
+      - rules:
+          - apiGroups:
+              - console.openshift.io
+            resources:
+              - consoleclidownloads
+            verbs:
+              - create
+              - delete
+              - deletecollection
+              - get
+              - list
+              - patch
+              - update
+              - watch
+          - apiGroups:
+              - camel.apache.org
+            resources:
+              - '*'
+            verbs:
+              - '*'
+        serviceAccountName: camel-k-operator
+    deployments:
+      - name: camel-k-operator
+        spec:
+          replicas: 1
+          selector:
+            matchLabels:
+              name: camel-k-operator
+          strategy:
+            type: Recreate
+          template:
+            metadata:
+              labels:
+                app: camel-k
+                camel.apache.org/component: operator
+                name: camel-k-operator
+            spec:
+              containers:
+              - command:
+                - kamel
+                - operator
+                env:
+                - name: WATCH_NAMESPACE
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.annotations['olm.targetNamespaces']
+                - name: OPERATOR_NAME
+                  value: camel-k
+                - name: POD_NAME
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.name
+                - name: NAMESPACE
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.namespace
+                image: docker.io/apache/camel-k:1.3.0
+                imagePullPolicy: IfNotPresent
+                livenessProbe:
+                  httpGet:
+                    path: /healthz
+                    port: 8081
+                  initialDelaySeconds: 5
+                  periodSeconds: 5
+                name: camel-k-operator
+                ports:
+                  - containerPort: 8080
+                    name: metrics
+                resources: {}
+              serviceAccountName: camel-k-operator
+      permissions:
```

Comment:
Can you remove 2 spaces in front of `permissions`?
```
-      permissions:
+    permissions:
```

Reply:
Should be fixed

Reply:
fixed